specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1797 @@
+                "type": "string"
+              },
+              "Azure-AsyncOperation": {
+                "description": "URL for checking the ongoing status of the operation.\nTo get the status of the asynchronous operation, send a GET request to the URL in Azure-AsyncOperation header value.\n\nFor more info: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#asynchronous-operations",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/OperationsStatus"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "GroupQuotaLimits"
+        ],
+        "summary": "Creates/Updates a new Group Quota request for a specific resources.",
+        "description": "Creates/Updates a new Group Quota request for a specific resources. A groupQuotaRequestId will be returned by the Service. The status will be updated periodically. The status Async polling is using standards defined at - https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#asynchronous-operations. Use the OperationsStatus URI provided in Azure-AsyncOperation header, the duration will be specified in retry-after header. Once the operation gets to terminal state - Succeeded | Failed,then the URI will change to Get URI and full details can be checked.",
+        "operationId": "GroupQuotaLimits_Update",
+        "x-ms-examples": {
+          "GroupQuotaLimits_Patch_Request_ForCompute": {
+            "$ref": "./examples/GroupQuotaLimits/PatchGroupQuotaLimits-Compute.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/ScopeInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "#/parameters/ResourceInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "groupQuotaRequest",
+            "in": "body",
+            "description": "The GroupQuota Request.",
+            "schema": {
+              "$ref": "#/definitions/GroupQuotaLimit"
```

Comment:
>GroupQuotaLimit [](http://example.com/codeflow?start=37&length=15)

Is every property of GroupQuotaLimit patch-able? If not, only add the patch-able properties to the Patch request payload.

Reply:
The Patch request is included, as per ARM requirements.  The QuotaRequest can be submitted using PUT or Patch APIs, as per ARM guidelines. It exists today for Subscription level QuotaRequests. Once the request is submitted, it cannot be modified.  
If you check the API, the Patch request is same as PUT, there is no requestID to be updated.

Reply:
Like mentioned on the other comment, PATCH will usually use a different definition, just to not include properties like "id".

Reply:
so we're extending the v5 common type for `ProxyResource` which marks id, name, type as readonly. is there any further action we need to do here?